src/Workspaces/CSharp/Portable/Extensions/MemberDeclarationSyntaxExtensions.cs

```diff
@@ -293,6 +293,8 @@ public static ArrowExpressionClauseSyntax GetExpressionBody(this MemberDeclarati
                     return ((OperatorDeclarationSyntax)memberDeclaration).ExpressionBody;
                 case SyntaxKind.ConversionOperatorDeclaration:
                     return ((ConversionOperatorDeclarationSyntax)memberDeclaration).ExpressionBody;
+                case SyntaxKind.ConstructorDeclaration:
+                    return ((ConstructorDeclarationSyntax)memberDeclaration).ExpressionBody;
```

Comment:
are destructors something we need to handle as well?

Reply:
Good point. I'd say so. I'll add another test case, and handle that.

Reply:
BTW I also noticed that this entire `switch` can be condensed into the following, however, from the other code in that file, this doesn't looks like something that gets done. Is this kind of refactoring appreciated/encouraged, or rather frown upon?

````csharp
switch (memberDeclaration.Kind())
{
    case SyntaxKind.MethodDeclaration:
    case SyntaxKind.OperatorDeclaration:
    case SyntaxKind.ConversionOperatorDeclaration:
    case SyntaxKind.ConstructorDeclaration:
    case SyntaxKind.DestructorDeclaration:
        return true;
}
````

Reply:
> Is this kind of refactoring appreciated/encouraged, or rather frown upon?

I would have to see the change suggested.  Based on that, i could tell you.  Worst case, you simply revert that change in the PR. :)

Reply:
Ah, ok 😅 Well, this then: [9262ad4](https://github.com/dotnet/roslyn/pull/33462/commits/9262ad4076c5f7c85fee42f94fe2a5a6f2b09431).

Please have another look.